packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -69,42 +69,41 @@ class AccountWizard extends React.Component {
         this.props.fetchStatesList();
         const { cancel, promise } = makeCancellablePromise(this.props.fetchResidenceList());
         this.cancel = cancel;
+        const { cancel: cancelFinancialAssessment, promise: financial_assessment_promise } = makeCancellablePromise(
+            this.props.fetchFinancialAssessment()
+        );
+        this.cancelFinancialAssessment = cancelFinancialAssessment;
+
+        Promise.all([promise, financial_assessment_promise]).then(() => {
+            this.setState({
+                items: getItems(this.props),
+                mounted: false,
+            });
 
-        promise
-            .then(() => {
+            if (!this.residence_list?.length) {
+                const items = this.state.items.slice(0);
+                this.getCountryCode().then(phone_idd => {
+                    if ('phone' in items[1].form_value) {
+                        items[1].form_value.phone = items[1].form_value.phone || phone_idd || '';
```

Comment:
or maybe extract it into named readable variable , for eg, const default_phone_number = items[1].form_value.phone